### ISSUE TITLE
[High contrast] Buttons always visually appear that they are in the rest state

### ISSUE THREAD

**Issue by jayrenn** [winjs/winjs#1346]
The buttons do not appear to change when in other states. They always look like they’re in the rest state. Should be:
![image](https://cloud.githubusercontent.com/assets/7266075/9261948/42e20e74-41c6-11e5-9403-0555f14ccea8.png)

**Comment by rigdern** [comment]
dupe of #1259.
